plugins/tweak/tweaks/partial-items.h

```diff
@@ -0,0 +1,99 @@
+// When displaying the names of partially-consumed items, show the percentage remaining
+// Potentially useful for revealing why random pieces of cloth or thread aren't suitable for jobs
+
+struct partial_items_hook_1 : df::item_barst {
```

Comment:
I would prefer eliminating the copy and paste by adding another macro that can be called like

```cpp
// Defines partial_items_hook_bar and calls IMPLEMENT_VMETHOD_INTERPOSE() with it
DEFINE_PARTIAL_ITEM_TWEAK(bar, 150);
// ...
DEFINE_PARTIAL_ITEM_TWEAK(cloth, 1000);
```